packages/xo-web/src/xo-app/backup/new/new-schedule.js

```diff
@@ -136,7 +136,17 @@ const New = decorate([
         )}
         <FormGroup>
           <label>
-            <strong>{_('healthCheck')}</strong>{' '}
+            <strong>
+              <a
+                className='text-info'
+                rel='noreferrer'
```

Comment:
```suggestion
                rel='noopener noreferrer'
```

Reply:
I think noopener is always true if noreferrer is true 
https://developer.mozilla.org/fr/docs/Web/HTML/Attributes/rel/noreferrer

Reply:
We need to remove a few `noopener`s then! 😬